src/main/kotlin/lotto/Lotto.kt

```diff
@@ -0,0 +1,66 @@
+package lotto
+
+import kotlin.collections.shuffled
+
+class Lotto(val lottoNumbers: List<LottoNumber>) {
+    init {
+        require(lottoNumbers.count() == 6)
+        require(isNotDuplicated(lottoNumbers))
+    }
+
+    fun getLottoNumbersAsInt(): List<Int> {
+        return lottoNumbers.map { it.value }
+    }
```

Comment:
If we start to utilize Kotlin more and more, we'll see there's so many simple-looking-ways of achieving the same goal :

```kotlin
val lottoNumbersAsInt: List<Int>
    get() = lottoNumbers.map { it.value }
```